### ISSUE TITLE
broken data export window

### ISSUE THREAD

**Issue by the-Arioch** [joyfullservice/msaccess-vcs-addin#531]
the "update" button in 3.4.23 was not very intuitive and was not very comfort - but it worked

the last vertion made something quite weird and outright counterintuitive - changing the hidden options without warning

![изображение](https://github.com/user-attachments/assets/686618d8-1859-4559-8a4e-c03199b9aeb2)

**Comment by joyfullservice** [comment]
I agree that this isn't the most intuitive approach. I am open to suggestions on how to improve this from a usability standpoint. 

For existing tables it is easy enough to find them in the list and set the export format, but sometimes you want to set up your options to always export a specific table name by default in your general options, without this table necessarily existing in the current database.

For example, I would expect that most users would definitely want custom ribbons exported by default, but the `USysRibbons` table may not exist in every database. If you are updating your global options, you probably want to have the add-in export this table data anytime it exists. The approach I took on this was to allow this entry in `vcs-options.json` to exist, even though no table by that name may exist in the current database.

![image](https://github.com/user-attachments/assets/766ac2c0-e874-49da-b06a-fcf3d5acdfc9)

The question is how do we intuitively manage these entries for tables that don't necessarily exist in the current database.

**Comment by the-Arioch** [comment]
some button to add special ros to the tables list, like "USysRibbons?" in the "nullable variable" fashion, meaning the table may or may not exist?

OTOH, do we really need it as a global preference?

I mean, the exact list of tables is clearly per-database thing, not per-computer.

Personally i consider that window to be an option to the particular export operation of a particular DB file. And i believe this kind of fine-grained settings is needed.

I think, hence, the "global defaults" should rather be a separate window. Which would address not specific tables (there are none, globally), but "kinds" of table (system, user-created, etc) and maybe name templates (filename-like or regex).

But whatever "global" settings are, when the database file is being opened they are recalculated into the local, per-table-name settings, when can be finely adjusted for the given export operation.
